daemons/fenced/fenced_remote.c

```diff
@@ -1836,27 +1810,26 @@ request_peer_fencing(remote_fencing_op_t *op, peer_device_info_t *peer,
             }
         }
 
-        // This is the only case in which result will be used
-        CRM_CHECK(result != NULL, return);
-
         if (op->state == st_query) {
             crm_info("No peers (out of %d) have devices capable of fencing "
                      "(%s) %s for client %s " CRM_XS " state=%s",
                      op->replies, op->action, op->target, op->client_name,
                      stonith_op_state_str(op->state));
 
-            pcmk__reset_result(result);
-            pcmk__set_result(result, CRM_EX_ERROR, PCMK_EXEC_NO_FENCE_DEVICE,
-                             NULL);
+            pcmk__reset_result(&op->result);
```

Comment:
Call is from a previous commit but seems redundant unless we need to free the stdout/stderr right now.

Reply:
Yep, just in case stdout/stderr is ever there